apisix/plugins/openwhisk.lua

```diff
@@ -30,6 +30,7 @@ local schema = {
         },
         service_token = {type = "string"},
         namespace = {type = "string", maxLength = 256},
+        package = {type = "string", maxLength = 256},
```

Comment:
Do we need to add a regex pattern to validate the input, like not `/` can be used in the package name?

Reply:
added regex validation according to [https://github.com/apache/openwhisk/blob/master/docs/reference.md#entity-names](https://github.com/apache/openwhisk/blob/master/docs/reference.md#entity-names)